core/nodepool/config/templates/stack-template.json

```diff
@@ -348,7 +348,7 @@
                   "Effect": "Allow",
                   "Resource":
                     { "Fn::Join": [ "", [
-                      "arn:aws:cloudformation:",
+                      "arn:aws{{if .IsChinaRegion}}-cn{{end}}:cloudformation:",
```

Comment:
Could be changed to `arn:{{$.Region.Partition}}:cloudformation:` if https://github.com/coreos/kube-aws/pull/390/files#r105572866 was addressed